resources/dicts/patrols/desert/training/any.json

```diff
@@ -153,7 +153,7 @@
                 "weight": 5
             },
             {
-                "text": "s_c decides there has to be a smarter way of figuring out if a badger has moved in, one that doesn't involve sticking their head in front of a predator. They lead the patrol on a search of the scrubland around the sett, and come up with fresh badger scat, stinky, horrible, and perfect proof of the new occupant.",
+                "text": "s_c decides there has to be a smarter way of figuring out if a badger has moved in, one that doesn't involve sticking their head in front of a predator. They lead the patrol on a search of the scrubland around the sett, and come up with fresh badger scat, stinky, horrible, and incontrovertible of the new occupant.",
```

Comment:
I'm a little wary of "incontrovertible" being too large a word.  I'll admit, I don't think I'd actually ever seen it before xD though I do understand the meaning.  Generally, though, we've tried not to get too complex with our vocabulary (it's a fine line, admittedly, I remember we had quite the discussion about "antagonize" as a button).

Reply:
could use @Tiritirimatangi's opinion on the matter tbh

Reply:
Oh, I get that for sure. I can just revert it to "perfect proof!" I'm also realizing I mistakenly deleted the work "proof," so I should fix that either way.

Reply:
Lol I don't mind either way occasionally I use weird word choices XD

Reply:
❤️❤️ I did decide to switch it back to "perfect proof." everything should be good to go now!